File: MODUL.6/PRAK605_2310817120010_NurHikmah.py
print("Input")
n = int(input())

print("Matriks A")
matriksA = []
for i in range(n):
    row = list(map(int, input().split()))
    matriksA.append(row)

print("Matriks B")
matriksB = []
for i in range(n):
    row = list(map(int, input().split()))
    matriksB.append(row)

# Menghitung hasil perkalian matriks
hasil = [[0 for _ in range(n)] for _ in range(n)]
for i in range(n):
    for j in range(n):
        for k in range(n):
            hasil[i][j] += matriksA[i][k] * matriksB[k][j]

# hasil perkalian matriks
print("\nOutput\nMatriks AXB")
for i in range(n):
    for j in range(n):
        print(hasil[i][j], end=" ")
    print()

File: MODUL.6/PRAK605_2310817120010_NurHikmah.c
#include <stdio.h>

int main()
{
    int n;

    // Membaca ordo matriks
    printf("Input\n");
    scanf("%d", &n);

    int matriksA[n][n];
    printf("Matriks A\n");
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            scanf("%d", &matriksA[i][j]);
        }
    }

    int matriksB[n][n];
    printf("Matriks B\n");
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            scanf("%d", &matriksB[i][j]);
        }
    }

    // Menghitung hasil perkalian matriks
    int hasil[n][n];
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            hasil[i][j] = 0;
            for (int k = 0; k < n; k++)
            {
                hasil[i][j] += matriksA[i][k] * matriksB[k][j];
            }
        }
    }

    // Menampilkan hasil perkalian matriks
    printf("\nOutput\nMatriks AXB\n");
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            printf("%d ", hasil[i][j]);
        }
        printf("\n");
    }
    return 0;
}
